backend/config/routes.rb

```diff
@@ -2,6 +2,7 @@
   namespace :admin do
     get '/search/users', to: "search#users", as: :search_users
     get '/search/products', to: "search#products", as: :search_products
+    post "/quick-switch", to: "quick_switch#find_object", as: :quick_switch
```

Comment:
The only issue that I have is that it returns a redirect. It seemed odd to me to have a `GET` change your location but I'll gladly change it if it makes more sense.